examples/gstreamer-receive/jsfiddle/demo.js

```diff
@@ -12,8 +12,10 @@ var log = msg => {
 }
 
 navigator.mediaDevices.getUserMedia({ video: true, audio: true })
-  .then(stream => pc.addStream(document.getElementById('video1').srcObject = stream))
-  .catch(log)
+  .then(stream => {
+    pc.addStream(document.getElementById('video1').srcObject = stream)
+    pc.createOffer().then(d => pc.setLocalDescription(d)).catch(log)
```

Comment:
Add something along the lines of:
```js
document.getElementById('localSessionDescription').value = btoa(JSON.stringify(d))
```
That way SD will be displayed right away in the browser

Reply:
Hey @svranesevic thanks for the review :)

The SD isn't complete yet, we are still waiting for ICE to finish. If we populate here we could generate an offer with no candidates (and Pion would have nothing to connect too). It might coincidentally work if things were fast enough or something blocked JS elsewhere though!

When `onicecandidate` returns NULL it means we have all our candidates [MDN](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/onicecandidate#Value)

Reply:
Hey @svranesevic if everything looks good would you be up for signing-off on this?

I would love to get this merged so other people don't run into the same issues. Our CI just doesn't let me merge without one review.

thanks!

Reply:
Hey @Sean-Der, sure!